components/board/board.go

```diff
@@ -97,9 +88,6 @@ type Board interface {
 type LocalBoard interface {
 	Board
 
-	// SPIByName returns an SPI bus by name.
-	SPIByName(name string) (SPI, bool)
-
```

Comment:
Can we follow up with clean up to the revolution pi and periph board modules? They are returning nil nil for these interfaces.

Reply:
Those modules shouldn't be touched until after this code is deployed. I'll file Jira tickets for later.

Reply:
I filed https://viam.atlassian.net/browse/RSDK-5776 and https://viam.atlassian.net/browse/RSDK-5777.